src/main/java/com/gooddata/http/client/LoginSSTRetrievalStrategy.java

```diff
@@ -76,34 +82,20 @@ public String obtainSst() throws IOException {
             if (status != HttpStatus.SC_OK) {
                 throw new GoodDataAuthException("Unable to login: " + status);
             }
-            final String sst = extractSST(response);
+            final String sst = TokenUtils.extractSST(response);
             if (sst == null) {
                 throw new GoodDataAuthException("Unable to login. Missing SST Set-Cookie header.");
```

Comment:
this is probably no longer true? As far as I understand now the SST is extracted from body. The second thing is - couldn't this check for not extracted SST moved to `TokenUtils`.?